### PR TITLE
Add feeds to users Home page

### PR DESCRIPTION
**Pull request by kian1213**
Hi sir, please review my pull request. Thanks po

### CODE REVIEW

**Comment by codegracie** [comment]
Okay

File: app/assets/stylesheets/styles.scss [hunk 1]
```diff
@@ -2,6 +2,15 @@ ul {
   list-style-type: none;
 }
 
+hr {
+    display: block;
```
Comment (dieunb) [inline app/assets/stylesheets/styles.scss]: should follow your style in this css file

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -27,9 +27,14 @@
             <ul>
               <% @activities.each do |activity| %>
                 <li>
-                  <%= activity_image_tag activity.id %>
-                  <%= activity.content %>
+                  <%= activity_image_tag activity.log_id %>
+                  <%= user_full_name activity.log_id %>
```
Comment (dieunb) [inline app/views/home/index.html.erb]: try `activity.log`

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -18,7 +18,13 @@ def datetime date
     I18n.l date, format: :short
   end
 
+  def user_full_name id
+    @user = User.find(id)
```
Comment (MuhammadTamzid) [inline app/helpers/application_helper.rb]: remove ()

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -18,7 +18,13 @@ def datetime date
...
+    link_to @user.full_name, @user
+  end
+
   def activity_image_tag id
-    image_tag User.find(id).avatar, size: "50x50"
+    @user = User.find(id)
```
Comment (MuhammadTamzid) [inline app/helpers/application_helper.rb]: remove ()

File: app/views/home/index.html.erb [hunk 5]
```diff
@@ -27,9 +27,14 @@
             <ul>
               <% @activities.each do |activity| %>
                 <li>
-                  <%= activity_image_tag activity.id %>
-                  <%= activity.content %>
+                  <%= activity_image_tag activity.log_id %>
```
Comment (MuhammadTamzid) [inline app/views/home/index.html.erb]: try activity.log

File: app/helpers/sessions_helper.rb [hunk 6]
```diff
@@ -29,6 +29,15 @@ def logged_in?
     !current_user.nil?
   end
 
+  def redirect_back_or default
+    redirect_to(session[:forwarding_url] || default)
```
Comment (MuhammadTamzid) [inline app/helpers/sessions_helper.rb]: remove ()

File: app/models/user.rb [hunk 7]
```diff
@@ -59,6 +59,13 @@ def send_activation_email
     UserMailer.account_activation(self).deliver_now
   end
 
+  def feeds
+    following_ids = "SELECT followed_id FROM relationships
+      WHERE  follower_id = :user_id"
+    Activity.where("log_id IN (#{following_ids})
```
Comment (dieunb) [inline app/models/user.rb]: better than if you move it to scope. You can refactor it on the next pull

**Pull request merged**